website/docs/d/datasource_cloudfunctions_function.html.markdown

```diff
@@ -45,11 +45,10 @@ exported:
 * `description` - Description of the function.
 * `available_memory_mb` - Available memory (in MB) to the function.
 * `timeout` - Function execution timeout (in seconds).
+* `runtime` - The runtime in which the function is running.
 * `entry_point` - Name of a JavaScript function that will be executed when the Google Cloud Function is triggered.
```

Comment:
Name of function that will be executed*

Reply:
You're right, thanks for the feedback! https://github.com/GoogleCloudPlatform/magic-modules/pull/915

Reply:
Glad to help! Awaiting this feature in the next release 🙃